src/main/java/com/Kh033Java/travelbook/repository/CountryRepository.java

```diff
@@ -12,8 +12,10 @@
 	
     List<Country> findAll();
     
-//    @Query("Match (c:Country)<-[v:VISITED]-(p:User) where c.name={countryName} return c,p")
     @Query("MATCH (n) WHERE n.name={countryName} RETURN n")
     Country getCountryByName(@Param("countryName") String countryName);
+
+    @Query("match (c:Country)<-[v:VISITED]-(u:User)-[cr:CREATED]->(n:Note) where u.login={userLogin} and n.isPublic=true return c")
+	List<Country> getCountriesWhereNotesPublic(@Param(value = "userLogin") String userLogin);

```

Comment:
change `[cr:CREATED]` to `[cr:CREATED_NOTE]`. look at User entity for naming relationships

Reply:
done